front_end/src/components/history/HistoryOverview.js

```diff
@@ -1,26 +1,26 @@
+// TODO:
+// - IF CHANGE, SHOW OLD/NEW DATES
+
 import React, { Component } from "react";
 import { withStyles, withTheme, withWidth } from "@material-ui/core/styles";
```

Comment:
withTheme and withWidth were imported but never used.

Reply:
removed